### ISSUE TITLE
Support Accessibility regarding resize text

### ISSUE THREAD

**Issue by tonglingchen** [SeleniumHQ/selenium-ide#975]
## 🐛 Bug Report

Fulfill the **W3 Standards 1.4.4 - Resize text**.

More info here: [https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html](https://www.w3.org/WAI/WCAG21/Understanding/resize-text.html)

## To Reproduce

Zoom to 200% and some controls are out of view port. 
![selenium](https://user-images.githubusercontent.com/22808540/75857884-e621ff80-5dab-11ea-9407-6b2252cf9a30.png)

## Expected behavior

When overflow happens, there should be a way to scroll to all controls. 

## Project file reproducing this issue (highly encouraged)

This is a issue with the UI, not of a specific Selenium scenario.

Issues without a reproduction project are likely to stall.

## Environment

OS: Windows 10
Selenium IDE Version: 3.16.1

**Comment by MarkoMackic** [comment]
Long test names can be horizontally scrolled ( referenced in #987 )

**Comment by tourdedave** [comment]
Fixed in [v3.17.0](https://github.com/SeleniumHQ/selenium-ide/releases/tag/v3.17.0)
